jscomps/LoginManagerPrompter.js

```diff
@@ -220,6 +220,12 @@ LoginManagerPromptFactory.prototype = {
               self.log(
                 "_doAsyncPrompt:run bypassed, UI is not available in this context"
               );
+            } else if (
+              e instanceof Components.Exception &&
+              e.result == Cr.NS_ERROR_FAILURE
```

Comment:
```suggestion
              e.result === Cr.NS_ERROR_FAILURE &&
              e.message === "This login already exists."
```
Also worth adding a reference to https://github.com/sailfishos-mirror/gecko-dev/blob/esr78/toolkit/components/passwordmgr/LoginHelper.jsm#L1305